commands/publish/README.md

```diff
@@ -154,6 +155,14 @@ lerna publish --canary --preid next
 When run with this flag, `lerna publish --canary` will increment `premajor`, `preminor`, `prepatch`, or `prerelease` semver
 bumps using the specified [prerelease identifier](http://semver.org/#spec-item-9).
 
+### `--pre-dist-tag <tag>`
```

Comment:
I don't see any of these flags documented in the yargs command builder for publish: https://github.com/lerna/lerna/blob/master/commands/publish/command.js

I suppose it's not vital, since yargs seems perfectly happy to consume them implicitly...